python/artm/batches_utils.py

```diff
@@ -205,8 +202,8 @@ def __reset_batch():
             for key in global_vocab.keys():
                 global_vocab[key][2] = False  # all tokens haven't appeared in this item yet
 
-            for token_id, value in enumerate(column):
-                value = value.item()
+            col = column if isinstance(column, type(np.zeros([0]))) else column.tolist()[0]
+            for token_id, value in enumerate(col):
```

Comment:
It is hard for me to review the change in ``_parse_nwd`` -- I've never used this format myself. Could you point me to an example of how this function is used? Is it covered by unit tests?